core/modules/utils/dom/dom.js

```diff
@@ -281,5 +281,48 @@ exports.getLocationPath = function() {
 	return window.location.toString().split("#")[0];
 };
 
+/*
+Collect DOM variables
+*/
+exports.collectDOMVariables = function(selectedNode,domNode,event) {
+	var variables = {},
+	    selectedNodeRect,
+	    domNodeRect;
+	if(selectedNode) {
+		$tw.utils.each(selectedNode.attributes,function(attribute) {
+			variables["dom-" + attribute.name] = attribute.value.toString();
+		});
+		//Add a variable with a popup coordinate string for the selected node
```

Comment:
It's a really tiny thing, but consistency matters – we include a space between the `//` and the start of the text of the comment

Reply:
done @Jermolene , thank you!